roughpy_jax/src/cpu/dense_ft_exp.h

```diff
@@ -0,0 +1,16 @@
+#ifndef ROUGHPY_JAX_CPU_DENSE_FT_EXP_H
+#define ROUGHPY_JAX_CPU_DENSE_FT_EXP_H
+
+#include <xla/ffi/api/c_api.h>
+
+// namespace rpy {
+// namespace jax {
+// namespace cpu {
+
+XLA_FFI_Error* cpu_dense_ft_exp(XLA_FFI_CallFrame*);
```

Comment:
@inakleinbottle how would you feel about instead rolling all of these headers into one header file?

Alternatively, as they are only used once, they could just be declared at the point of use in roughpy_jax/src/roughpy_jax_module.c.

Reply:
I don't have a strong preference. keeping them separate might have some "cognitive overhead" advantages over combined - it makes it more obvious where things are defined.